### PR TITLE
Fix emmiting errors

### LINKED ISSUE #47: emitErrors does not work anymore
Cucumber cli returns following object when

    { shouldExitImmediately: false, success: true }

Due to this update condition is always evaluated to true and does not thorw errors when test fails

      cli.run().then((success) => {
        if (success || !emitErrors) {

I have checked difference between versions and this change was introduced in version 4 of cucumber and it was not working properly since.

### PR DESCRIPTION
**Pull request by maciejmaciejewski**
Resolves #47 
Applied change for cucumber 4.X and higher with maintained backward compatibility.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -71,7 +71,8 @@ module.exports = function ({
             stdout: process.stdout,
         });
 
-        cli.run().then((success) => {
+        cli.run().then((result) => {
+            const success = typeof(isSucceresultssNew) === 'object' ? result.success : result
```
Comment (vgamula) [inline index.js]: @maciejmaciejewski is this a typo or I'm missing something? `isSucceresultssNew ` 👀

**Review by vgamula — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (maciejmaciejewski) [reply index.js]: Indeed, fixed it

**Review by vgamula — APPROVED** [review]

**Comment by vgamula** [comment]
@maciejmaciejewski, thanks for your contribution 🙌 
I will release the new version to npm shortly

**Comment by vgamula** [comment]
Published v0.0.24 to NPM

**Pull request merged**